projects/plugins/jetpack/_inc/client/writing/composing.jsx

```diff
@@ -203,6 +203,16 @@ export class Composing extends React.Component {
 						{ __( 'Discover Jetpack tools in the block editor', 'jetpack' ) }
 					</CompactCard>
 				</>
+			),
+			aiAssistantLink = (
```

Comment:
It may be worth adding an additional `showMyJetpack` check in here, since My Jetpack is not always available to folks.
https://github.com/Automattic/jetpack/blob/f94dc3004d5cd36a2e2a2ad429eb48d8a997471a/projects/plugins/jetpack/_inc/client/state/initial-state/reducer.js#L445-L451

Reply:
Thanks @jeherve, I'll see to it!